runtime/src/accounts.rs

```diff
@@ -800,11 +841,16 @@ impl Accounts {
             }
             let (res, hash_age_kind) = &res[i];
             let maybe_nonce = match (res, hash_age_kind) {
-                (Ok(_), Some(HashAgeKind::DurableNonce(pubkey, acc))) => Some((pubkey, acc)),
+                (Ok(_), Some(HashAgeKind::DurableNonceFull(pubkey, acc, maybe_fee_account))) => {
+                    Some((pubkey, acc, maybe_fee_account))
```

Comment:
til: it looks like this can be None. That's because we don't use `maybe_nonce` anymore if `res` is `Ok(_)`, considering we enabled `consistent recentblockhashes sysvar`.

Not that I'm asking you to do now with the follow up pr (#13868). just a note to solidify my understanding.